packages/ui/src/primitives/tailwind/Input/index.tsx

```diff
@@ -79,7 +79,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
 
     const twClassname = twMerge(
       'text-base font-normal tracking-tight',
-      'textshadow-sm flex h-9 w-full rounded-lg border border-theme-primary bg-theme-surfaceInput px-3.5 py-5 transition-colors',
+      'textshadow-sm flex h-9 w-full rounded-lg border border-theme-primary bg-theme-surfaceInput px-2 py-5 transition-colors',
```

Comment:
The [native designs](https://www.figma.com/design/Gn8EScrMmg1g6sQBy8f6lz/Hi-FI%3A-Dashboard-%26-Admin?node-id=863-101508&t=sPnUqPsUxXN1kAiT-0) in input having a 14px padding.